pkg/kubelet/volumes.go

```diff
@@ -77,12 +77,18 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.BoundPod) (volumeMap, error) {
 	for i := range pod.Spec.Volumes {
 		volSpec := &pod.Spec.Volumes[i]
 
+		podRef, err := api.GetReference(pod)
```

Comment:
@lavalamp can you provide more clarity?  I'm afraid I don't understand.

The code as written above fails because BoundPod does not have a refLink.   If you're saying I can use an ObjectRef safely here, how would I do that?